Backend/core/db.py

```
@@ -1,119 +1,288 @@
+
+def _insert_timeseries(data: Dict[str, Any], timestamp: int):
+    """Insert data using Redis TimeSeries (local mode)."""
+    for key, value in data.items():
+        try:
+            if isinstance(value, bool):
+                value = 1 if value else 0
+            elif isinstance(value, bytes):
+                value = ord(value)
+            
+            r.execute_command('TS.ADD', key, timestamp, value, 'ON_DUPLICATE', 'LAST')
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)